core/src/test/java/org/apache/accumulo/core/client/admin/NewTableConfigurationTest.java

```diff
@@ -93,4 +107,75 @@ public void testCreateOffline() {
     NewTableConfiguration ntcOnline = new NewTableConfiguration();
     assertTrue(ntcOnline.getInitialTableState() == InitialTableState.ONLINE);
   }
+
+  public void populateOptions() {
+    options = new HashMap<>();
+    options.put("hasher", "murmur3_32");
+    options.put("modulus", "5");
+  }
+
+  public void populateInvalidOptions() {}
+
+  /**
+   * Verify enableSampling returns
+   */
+  @Test
+  public void testEnableSampling() {
+    SamplerConfiguration samplerConfig = new SamplerConfiguration("test");
+    NewTableConfiguration ntcSample1 = new NewTableConfiguration().enableSampling(samplerConfig);
+    assertTrue(ntcSample1.getProperties().containsValue("test"));
+
+    RowSampler rowSampler = new RowSampler();
```

Comment:
I think row sampler could be removed and the checks could be made stronger like the following.

```java
    SamplerConfiguration sha1SamplerConfig = new SamplerConfiguration("com.mysampler");
    sha1SamplerConfig.setOptions(options);
    NewTableConfiguration ntcSample2 =
        new NewTableConfiguration().enableSampling(sha1SamplerConfig);
    assertEquals("com.mysampler", ntcSample2.getProperties().get("table.sampler"));
    assertEquals("5", ntcSample2.getProperties().get("table.sampler.opt.modulus"));
    assertEquals("murmur3_32", ntcSample2.getProperties().get("table.sampler.opt.hasher"));
```